File: Samples-Xamarin/Fragments/Dataform/DataFormPlaceholderLayoutFragment.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Util;
using Android.Views;
using Android.Widget;

using Com.Telerik.Widget.Dataform.Visualization;
using Com.Telerik.Widget.Dataform.Engine;

namespace Samples
{
	public class DataFormPlaceholderLayoutFragment : Android.Support.V4.App.Fragment, ExampleFragment
	{
		public String Title() {
			return "Placeholder Layout";
		}

		public override View OnCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
			ViewGroup rootLayout = (ViewGroup)inflater.Inflate(Resource.Layout.fragment_dataform_grouping, container, false);

			RadDataForm dataForm = (RadDataForm)rootLayout.FindViewById(Resource.Id.data_form_grouping);
			dataForm.FillViewport = true;
			dataForm.LayoutManager = new DataFormPlaceholderLayoutManager(this.Activity, Resource.Layout.dataform_placeholder_layout);
			dataForm.SetEntity (new Person());

			return rootLayout;
		}
	}
}

File: Samples-Java/Samples/src/main/java/fragments/dataform/DataFormPlaceholderLayoutFragment.java
package fragments.dataform;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.telerik.android.sdk.R;
import com.telerik.widget.dataform.visualization.DataFormPlaceholderLayoutManager;
import com.telerik.widget.dataform.visualization.RadDataForm;

import activities.ExampleFragment;

public class DataFormPlaceholderLayoutFragment extends Fragment implements ExampleFragment {
    @Override
    public String title() {
        return "Placeholder Layout";
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        ViewGroup rootLayout = (ViewGroup)inflater.inflate(R.layout.fragment_dataform_grouping, container, false);

        RadDataForm dataForm = (RadDataForm)rootLayout.findViewById(R.id.data_form_grouping);
        dataForm.setFillViewport(true);
        dataForm.setLayoutManager(new DataFormPlaceholderLayoutManager(this.getActivity(), R.layout.dataform_placeholder_layout));
        dataForm.setEntity(new Person());

        return rootLayout;
    }
}
